templates/taxi/manufacturer_list.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
+    <h1>Manufacturer List <a href="{% url "taxi:manufacturer-list-create" %}" style="float:right">+</a>
```

Comment:
Ensure that the link to create a new manufacturer is correctly styled and functional. The '+' symbol is a good visual cue for adding new entries.